### ISSUE TITLE
Check if Tx shall be turned off when not used

### ISSUE THREAD

**Issue by dl1com** [HAMNET-Access-Protocol/HNAP4PlutoSDR#4]
The self reception of the TX-LO will distort our received signal: https://wiki.analog.com/university/tools/pluto/hacking/listening_to_yourself?s[]=adalm&s[]=pluto&s[]=hardware

We already generate a PTT signal in our client application which toggles a GPIO port. We should check wether its possible to also enable/disable the TX LO during PTT toggling. If disabling the LO completely has a too large delay, we could try to lower the transmit power of the TX instead of turning it off.

**Comment by DG9BFC** [comment]
yes ... switch power down to minus 78 dbm works ... but is that needed here??
the tx lo will only block rx performance if both are on same frequency but if rx and tx have a shift then there is no problem (with base station in full duplex with filters there is no problem so why should there be at user end??) ... if you implement single frequency packet radio (1k2, 9k6, aprs, whatever) then you just have to shift tx maybe 25 kc away while receiving ...
